sql/core/v2.3.4/src/main/java/org/apache/spark/sql/execution/datasources/orc/OrcColumnarBatchReader.java

```diff
@@ -0,0 +1,210 @@
+/*
```

Comment:
Were these files really added or just moved? is it possible to record this as a move in the git change if so?

Reply:
These files copy from `sql/core/v1.2.1` and rename some package names. I tried to add the history of the files but failed because git seems does not support copy file.

Reply:
Let's review it first: https://github.com/apache/spark/pull/24119